libs/CCAppCommon/include/CCAppCommon.h

```diff
@@ -1,26 +1,26 @@
-//##########################################################################
-//#                                                                        #
-//#                            CLOUDCOMPARE                                #
-//#                                                                        #
-//#  This program is free software; you can redistribute it and/or modify  #
-//#  it under the terms of the GNU General Public License as published by  #
-//#  the Free Software Foundation; version 2 or later of the License.      #
-//#                                                                        #
-//#  This program is distributed in the hope that it will be useful,       #
-//#  but WITHOUT ANY WARRANTY; without even the implied warranty of        #
-//#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the          #
-//#  GNU General Public License for more details.                          #
-//#                                                                        #
-//#          COPYRIGHT: CloudCompare project                               #
-//#                                                                        #
-//##########################################################################
+// ##########################################################################
```

Comment:
Any way to fix this shift in the header? I believe @bramburn had the same issue on his PR (https://github.com/CloudCompare/CloudCompare/pull/2026). By the way, maybe some things should be shared between the 2?

Reply:
That's the only things that strikes me, and it's not a big deal. It 'makes sense' in a way as it looks like comment lines...

Reply:
No, let's keep things simple, I'll live with a space in the header ;)

Reply:
Do you want to merge this PR first, or do you directly want to expand the scope of the clang format file?

Reply:
I think I want to expand the scope of the PR first